src/expr/proof_generator.h

```diff
@@ -105,6 +105,30 @@ class ProofGenerator
   virtual std::string identify() const = 0;
 };
 
+/**
+ * debug check closed on Trace c, context ctx is string for debugging
+ *
+ * @param reqGen Whether we consider a null generator to be a failure.
+ */
+void pfgEnsureClosed(Node proven,
```

Comment:
Please add to the doc above the policy for whether this method being a no-op. It's unclear as is.